### ISSUE TITLE
Incorrect member-delimiter-style error

### ISSUE THREAD

**Issue by iansan5653** [gajus/eslint-plugin-jsdoc#405]
The following comment gives a `member-delimiter-style` ("Expected a semicolon") on the word "to":
```js
  /** 
   * Action type to take. If the form submission deletes something, set this
   * to `"negative"`. Defaults to `"primary"` as form submission is the primary
   * action to take in a form.
   * 
   * @see {Button} for more info on this prop.
   */
```
![image](https://user-images.githubusercontent.com/2294248/67024946-36d1de00-f0d3-11e9-9a95-b38612397437.png)

**Comment by iansan5653** [comment]
Never mind, it disappeared when I reopened the file.
